docs/products/postgresql/howto/check-iop.rst

```diff
@@ -0,0 +1,57 @@
+IOP capability of plans
+-----------------------
+
+This article will explain what is the typical IOP capability used for common plans in pg:
+
+Hobbyist
+''''''''
+
+==============================     ================     ===================     ================
```

Comment:
Here is an example how to add tables that are easy to maintain later: https://github.com/aiven/devportal/commit/806a65ce443775733651c6737c5b612d6b183ebe